routes/login/login.session.spec.js

```diff
@@ -0,0 +1,197 @@
+const session = require('supertest-session')
```

Comment:
Making this its own file was a good idea 👍

Reply:
Yeah, it was getting really complex :/